term-size.c

```diff
@@ -1,10 +1,9 @@
 #include <stdio.h>
-#include <string.h> // strerror
-#include <errno.h>  // errno
-#include <fcntl.h>  // open(), O_RDONLY
-#include <unistd.h> // close()
-#include <curses.h> // setupterm(), tigetnum()
-#include <term.h>   // ^ Same as above
+#include <string.h>    // strerror
+#include <errno.h>     // errno
+#include <fcntl.h>     // open(), O_RDONLY
+#include <unistd.h>    // close()
+#include <sys/ioctl.h> // ioctl()
 
 int main() {
 	int tty_fd = open("/dev/tty", O_RDONLY);
```

Comment:
I saw some code using `O_RDONLY|O_NONBLOCK`. Would it be beneficial? Or does it not matter?